test/Tester/project.json

```diff
@@ -10,7 +10,8 @@
     "WindowsAzure.Storage": "7.0.0",
     "xunit": "2.1.0",
     "xunit.runner.visualstudio": "2.1.0",
-    "Xunit.SkippableFact": "1.2.14"
+    "Xunit.SkippableFact": "1.2.14",
+    "System.Management.Automation.dll": "10.0.10586"
```

Comment:
Either we can move those utilities to another project, or if you don't want to take this work in this PR, you can do the same thing as TesterInternal, and have this new test project reference Tester in order to use `BaseTestClusterFixture`.
We can refactor `BaseTestClusterFixture` and some other extra stuff into their own project later. I believe that would be more appropriate instead of conflating that move in this PR.